roles/wptrunner/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install PIP
+  easy_install:
+    name: pip
+    state: present
+
+- name: Fetch project source
+  git:
+    repo: https://github.com/w3c/wptrunner.git
+    dest: ./wptrunner
+
+- name: Install dependencies
+  shell: |
+    cd wptrunner && \
+    pip install --editable ./
+  become: true
+
+- name: Create manifest file
+  shell: |
+    mkdir meta && \
```

Comment:
Should this directory be created with the download and tools directories in `roles/directories/tasks/main.yml`? I only ask because Ansible suggested it should. I have no horse in this race.